### PR TITLE
Fixes npm packages.json

### PR DESCRIPTION
**Pull request by MichellanneLi**
adds js hint to npm

### CODE REVIEW

**Comment by arthur24b6** [comment]
Could you change the homepage url to this repo?

**Comment by mathewpeterson** [comment]
@MichellanneLi Can you address Arthur's comment?

**Comment by MichellanneLi** [comment]
@mathewpeterson I'd be happy to, but could you please walk me through it? I haven't had the opportunity to work with Git since our meeting on Tuesday. Let me know when would be convenient for you.

**Comment by MichellanneLi** [comment]
https://github.com/palantirnet/butler/pull/10/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12

Should this be updated as well?

File: package.json [hunk 1]
```diff
@@ -16,5 +16,24 @@
   "bugs": {
     "url": "https://github.com/labyrwa/fed-automation/issues"
```
Comment (MichellanneLi) [inline package.json]: Should this line be updated as well?

Comment (mathewpeterson) [inline package.json]: Yes please! :)

**Comment by MichellanneLi** [comment]
Hi @mathewpeterson, I made the requested changes for your review. Thanks!

**Comment by mathewpeterson** [comment]
@labbydev Do these changes look OK? I would like to merge.

**Comment by labbydev** [comment]
@mathewpeterson these look good to me. I love to see progress here!!

**Comment by mathewpeterson** [comment]
:+1:

**Pull request merged**